packages/gestalt/src/Sheet.css

```diff
@@ -0,0 +1,66 @@
+@keyframes :global(g-sheet-slide-in-ltr) {
+  0% {
+    transform: translateX(100%);
+  }
+  
+  100% {
+    transform: translateX(0%);
+  }
+}
+
+@keyframes :global(g-sheet-slide-in-rtl) {
+  0% {
+    transform: translateX(-100%);
+  }
+
+  100% {
+    transform: translateX(0%);
+  }
+}
+
+@keyframes :global(g-sheet-slide-out-ltr) {
+  0% {
+    transform: translateX(0%);
+  }
+
+  100% {
+    transform: translateX(100%);
+  }
+}
+
+@keyframes :global(g-sheet-slide-out-rtl) {
+  0% {
+    transform: translateX(0%);
+  }
+
+  100% {
+    transform: translateX(-100%);
+  }
+}
+
+.container {
+  composes: fixed borderBox flex justifyEnd left0 top0 itemsCenter from "./Layout.css";
+  composes: xsCol12 from "./Column.css";
+  height: 100%;
+  z-index: 1000;
```

Comment:
In this case, both components are `<div>` because we're applying classes and hence can't use `<Box>` or `<Sticky>` which are the only components that can hold `zIndex` at the moment. 
So I'm not fully clear if should we necessarily add another `<Box>` wrapper only to be able to pass in a `zIndex` though JS and then use `FixedZIndex` or `CompositeZIndex`.

Any thoughts @AlbertCarreras @christianvuerings ?

Reply:
BTW this `z-index` can be removed, it was for testing purposes and I forgot to remove. But the one below can apply to this discussion.

Reply:
@themindfuldev having it hardcoded and set to a magic number `1000` is less than ideal. We should follow the same pattern as we do with `Modal`, where we do have this wrapper component if necessary.

In Pinterest's codebase, we will need a wrapper component already to stop impression logging + add the zIndex. Take a look at `app/common/react/ui/Modal.js` and `MODAL_ZINDEX`

Reply:
Hi Christian! Sorry this part didn't make it to this push but I first tried adding `zIndex` to `<Layer>` but it did not play well with `<StopScrollBehavior>` and `<TrapFocusBehavior>` so I'm trying with `<Box>` now. Thank you so much